frontend/src/containers/TheHeaderDropdownNotif.vue

```diff
@@ -155,6 +153,14 @@ export default {
         this.redirect(notif);
       }
     },
+    formattedDate(timestamp) {
+      var currDate = new Date(timestamp);
+      var numberOfMlSeconds = currDate.getTime();
+      var subFourHours = 4 * 60 * 60 * 1000; //GMT -4
+      var newDate = new Date(numberOfMlSeconds - subFourHours);
+      var newTime = newDate.toString().split(' GMT')[0];
+      return newTime.substring(0, newTime.indexOf(':') +3);
+    },
```

Comment:
Refer to my first comment. This is technically duplicated code that could be refactored.

Reply:
I have addressed this refactoring in this [commit](https://github.com/purplesmurf1998/Autotracks/pull/105/commits/4016c674218b60f0932c893a6c8dcec14266843c)